src/ResourceManager/OperationalInsights/Commands.OperationalInsights/DataSources/GetAzureOperationalInsightsDataSourceCommand.cs

```diff
@@ -57,7 +57,6 @@ public class GetAzureOperationalInsightsDataSourceCommand : OperationalInsightsB
         HelpMessage = "The data source name.")]
         [Parameter(ParameterSetName = ByWorkspaceObjectByKind)]
         [ValidateSet(
-            PSDataSourceKinds.AzureAuditLog,
```

Comment:
As I stated in PR description, this is well planned, we have disabled New-*AzureAuditLogDataSource since last Nov, aliasing to New-*AzureActivityLogDataSource, so AzureAuditLogDataSource is readOnly. 

From backend we have convert all existing AuditLog datasource to ActivityLog datasource.